dodo.py

```diff
@@ -0,0 +1,111 @@
+import glob
+import os
+import subprocess
+
+LOCALES = ["en","de", "es", "fr", "ga", "it", "ja", "ko", "pt", "sv", "zh"]
+
+def task_mo():
+    """generate a locales .mo file from .po file.   Use task locales_combine to produce final result"""
```

Comment:
I think this is the same as the code I moved into the file bin/i18n/pygettext.py?